pkgs/top-level/python-packages.nix

```diff
@@ -674,6 +674,10 @@ in {
 
   flufl_lock = callPackage ../development/python-modules/flufl/lock.nix { };
 
+  fluidasserts = callPackage ../development/python-modules/fluidasserts {
+    mixpanel' = callPackage ../development/python-modules/fluidasserts/overlays/mixpanel.nix { };
```

Comment:
just add the dependency to python-modules, want to avoid the case where this will be imported twice.